internal/generator/vector/output/config_strategy.go

```diff
@@ -53,6 +56,8 @@ func (o Output) VisitBuffer(b common.Buffer) common.Buffer {
 		switch o.spec.Tuning.Delivery {
 		case logging.OutputDeliveryModeAtLeastOnce:
 			b.WhenFull.Value = common.BufferWhenFullBlock
+			b.Type.Value = buffertTypeDisk
+			b.MaxSize.Value = minBufferSize
```

Comment:
is it correct, **Max**Size set to **min**BufferSize?

Reply:
yes.  we are not allowing users to actually set the on disk buffer size and 'minBufferSize' is the value required to use on disk buffering